submarine-sdk/pysubmarine/submarine/experiment/api/experiment_client.py

```diff
@@ -38,6 +39,21 @@ def __init__(self, host):
         api_client = ApiClient(configuration=configuration)
         self.experiment_api = ExperimentApi(api_client=api_client)
 
+        """
+        For user login
+        :return: Passed or not
+        """
+        l = ldap.initialize(‘ldap://172.16.1.163:389‘)
```

Comment:
`ldap://172.16.1.163:389`, is this the test address?
Where did this address come from? Is the LDAP service address? 
Do we need to configure it in the submarine config file?

Reply:
Yes , this is a test LDAP server address.
For those who want to use LDAP login , they should change the address by them self.